extensions/amp-experiment/0.1/variant.js

```diff
@@ -34,6 +36,12 @@ export function allocateVariant(win, experimentName, config) {
   assertName(experimentName);
   validateConfig(config);
 
+  // Variant can be overridden from URL fragment.
+  const override = viewerFor(win).getParam(ATTR_PREFIX + experimentName);
+  if (override && config.variants.hasOwnProperty(override)) {
```

Comment:
For stronger safety, I suggest doing this in `getVariantOverride_()` so invalid values never make it to the overrides map to start with.